index.js

```diff
@@ -8,12 +8,24 @@ const cheerio = require("cheerio");
 
 require("colors");
 
-
+const HELP = (process.argv.includes("-h") || process.argv.includes("--help"));
 const METRIC = process.argv.includes("--metric");
 const GRAMS_PER_OZ = 28.3495;
 const SFSPCA_BASE = "https://www.sfspca.org"
 const ADOPTION_PAGE = `${SFSPCA_BASE}/adoptions/cats`;
 
+if(HELP){
+  console.log("Some cats are born fat, some achieve fatness, others have fatness thrust");
```

Comment:
Just to make this a bit cleaner, could you define this string as a const with `\n`'s in it, and just call console.log once?